src/main/java/com/ericsson/ei/subscription/HttpRequestSender.java

```diff
@@ -69,16 +69,8 @@ public boolean postDataMultiValue(String notificationMeta, MultiValueMap<String,
                 response = rest.postForEntity(notificationMeta, request, JsonNode.class);
             }
 
-        } catch (HttpClientErrorException e) {
-            LOGGER.error("HTTP-request failed, bad request!\n When trying to connect to URL: " + notificationMeta
-                    + "\n " + e.getMessage());
-            return false;
-        } catch (HttpServerErrorException e) {
-            LOGGER.error("HTTP-request failed, internal server error!\n When trying to connect to URL: "
-                    + notificationMeta + "\n " + e.getMessage());
-            return false;
         } catch (Exception e) {
-            LOGGER.error(e.getMessage(), e);
+            LOGGER.error("HTTP-request failed when trying to connect to URL: {}", notificationMeta, e);
```

Comment:
Please revert those changes, a PR with more details in the exceptions exists

Reply:
I'll just fix the conflict with your PR.